showcase/src/assets/translations/en.json

```diff
@@ -154,6 +154,45 @@
         "SECTIONS": "Menu with sections",
         "TITLE": "Stark app menu"
       },
+      "MESSAGE_PANE": {
+        "BUTTONS": {
+          "CENTER": "Center",
+          "CHANGE_THEME": "Change Theme",
+          "CLEAR_AND_SHOW_NEW_MESSAGES": "Clear & Show New Messages",
+          "CLEAR_MESSAGES": "Clear Messages",
+          "LEFT": "Left",
+          "RIGHT": "Right",
+          "SHOW_MESSAGES": "Show Messages"
+        },
+        "DESCRIPTION": "The Message Pane will persist on the page until dismissed, but will not stop page interactions (like dialogs do). It can be used to show validation errors of the form, if your app needs to keep then until dismissed.",
```

Comment:
small typo issue: "... to keep the**m** until dismissed." 
But, I think we could change it for something like this:
The Message Pane will persist on the page until dismissed, but will not stop page interactions (like dialogs do). It can be used, for instance, to show validation errors of the form in case your app needs to keep them until dismissed.

What do you think ? 😊

Reply:
yeah, it is better ;) I'll adapt it